docusaurus.config.js

```diff
@@ -11,7 +11,7 @@ const config = {
   title: "Clearpath Robotics Documentation",
   url: "https://docs.clearpathrobotics.com",
   baseUrl: "/",
-  onBrokenLinks: "throw",
+  onBrokenLinks: "warn",

```

Comment:
We need to change this back to "throw" before merging

Reply:
Fixed in https://github.com/clearpathrobotics/cpr-documentation/pull/129